book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+# Collaborative Code Review
+
+## 1. Problem statement
+* Ensuring your code to be reviewed is accessible/reproducible (across different operating systems and coding setups)
+* Trying to learn a new language and not knowing the most efficient way to solve a given problem. Others might have better ways to solve that problem which you might not know about. 
+* Getting feedback about our code's interface, in order to make it more usable.
+* How do you get this feedback while being considerate of other people's time?
+* Getting started and gauging interest in a group code review
+
+## 2. Solutions & Recommendations
+* Code with comprehension and (try to)  make the code reviewing process easier by being clear and organised with the format and logic. The code itself does not have to look professional due to limited experience of the coder, but should be equipped with comments that help reviewers to understand. Good format to allow advanced coding (Object-oriented programming, list comprehension) to be applied.  
+* Be as specific as you can in terms of the aspects of your code that need review. Is it a specific function that you are worried about? Or perhaps the way you have structured your code?
+* Ask someone else to use your library/run your code, even for a toy example. If they find it hard or ask you questions on how to use it, then there is room for improvement :smile: 
+* Develop code with other operating systems in mind where possible (for example, specify your OS and environment settings) and include code run on binder to ensure others can run the code in some form - testing code on multiple operating systems also provides a better opportunity for reproducibility testing!
+* Contacting your team for gauging interest (for example, via email, raising the topic in lab meetings or sharing a poll) alongside your institution for any more experienced recommendations on how to get started with a group code review - for example, university admin teams may have setup similar groups before. Also include people interested and institution experts in your prelimnary meetings to setup the code review group.
+
+
+## 3. Examples
```

Comment:
this feels like it should be integrated with the previous section so that you walk people through the process of a code review step-by-step in the chapter: 
1. Things to keep in mind when coding in the first place
2. Getting started asking for help with code reviews
3. How to organise them so that you make the best use of everyone's time